### PR TITLE
Update NEWS file for 1.4.3rc1

### PR DESCRIPTION
**Pull request by jdinan**
Signed-off-by: James Dinan <james.dinan@intel.com>

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -8,6 +8,9 @@ v1.4.3rc1
 - Added proposed OpenSHMEM wait/test-any/some/all to shmemx interfaces.
 - Added proposed OpenSHMEM nonblocking fetching atomic operations to shmemx
   interfaces.
+- Added proposed OpenSHMEM SHMEM_CTX_INVALID constant to shmemx interfaces.
```
Comment (wrrobin) [inline NEWS]: Should we say `SHMEMX_CTX_INVALID`? Although we did mention as shmemx interface. Both ok, I guess.

Comment (jdinan) [reply NEWS]: Yeah, I went back and forth on that, too.

Comment (jdinan) [reply NEWS]: :man_shrugging:

Comment (wrrobin) [reply NEWS]: I am ok then with whatever you decide :)

**Pull request merged**